shell/platform/embedder/embedder.cc

```diff
@@ -2166,6 +2167,15 @@ FlutterEngineResult FlutterEngineInitialize(size_t version,
         "Could not infer the Flutter project to run from given arguments.");
   }
 
+  // Begin shorebird
+  if (args->shorebird_args.shorebird_yaml_contents) {
```

Comment:
if you make shorebird_args a struct then this will check that pointer instead.

Although maybe this already works fine, assuming all embedders use calloc (clearing alloc) to allocate the FlutterProjectArgs.  Or I guess if they allocate it on the stack then maybe C++ automatically zeros it?

Reply:
It looks like C++ does *not* zero-initialize structs on the stack. 🤦  So other users would end up with garbage, likely non-zero, values  in shorebird_args.

Reply:
We recommend zero initialization of structs. '= {};' should do the trick.

Reply:
This should be handled by `FlutterProjectArgs flutterArguments = {};` in `shell/platform/darwin/macos/framework/Source/FlutterEngine.mm`, correct?